MTA-6889 CLI refactored commands (#372)
migtools/mta-documentation: assemblies/cli-guide/assembly_about-the-command-line-tool.adoc

:_newdoc-version: 2.18.3
:_template-generated: 2025-06-24

ifdef::context[:parent-context-of-about-the-command-line-tool: {context}]

:_mod-docs-content-type: ASSEMBLY

ifndef::context[]
[id="about-the-command-line-tool"]
endif::[]
ifdef::context[]
[id="about-the-command-line-tool_{context}"]
endif::[]
= About the {ProductShortName} commands

:context: about-the-command-line-tool

[role="_abstract"]

> To align with industry standards command-line interface (CLI) patterns and for an improved developer experience, Migrators can use a structured command model comprising seven main commands.

Reviewer: ```suggestion
As a migrator, you can use a structured command model that consists of seven main commands to improve your experience and align with standard command-line interface (CLI) patterns.
```

Pronouns and Audience (IBM Style): Changed "Migrators" to "you" (and "your experience"). IBM Style dictates that when referring to the reader, you should use "you" instead of a third-person persona or role like "user" or "Migrator". This creates a more direct, active, and engaging tone.

Awkward Noun Strings (IBM Style): Changed "industry standards command-line interface (CLI) patterns" to "standard command-line interface (CLI) patterns". Stacking too many nouns ("industry standards") before the main subject makes the sentence difficult to read and translate.

Simple Words (IBM Style / CQA 2.1): Changed "comprising" to "that consists of". IBM Style and CQA 2.1 advocate for using simple, common words to ensure the text is easily understood by global audiences and easily parsed by machine translation.

User Intent (Red Hat SSG): Maintained the focus on what the user can accomplish ("...you can use a structured command model..."), which perfectly aligns with the Red Hat SSG guidelines for writing short descriptions.
